static/app/utils/discover/urls.tsx

```diff
@@ -27,6 +33,59 @@ export function eventDetailsRoute({
   return `/organizations/${orgSlug}/discover/${eventSlug}/`;
 }
 
+/**
+ * Return a URL to the trace view or the event details view depending on the
+ * feature flag.
+ *
+ * TODO Abdullah Khan: Add link to new trace view doc explaining why we route to the traceview.
+ */
+export function generateEventIDLinkTarget({
```

Comment:
```suggestion
export function generateTraceViewLink({
```
is this more accurate? It's not obvious what EventIDLinkTarget actually results in

Reply:
It doesn't always route to the traceview yet right. It determines the target of the event id links. Maybe that comment above it helps?

Reply:
I think it's fine as we are deprecating the eventID linking anyways, so in the future, we will just remove the legacy link handling and wont have to rename the method.